src/emmc.c

```diff
@@ -156,6 +156,12 @@ static gboolean r_emmc_force_part_write(const gchar *device, gchar value, GError
 	}
 
 	f = g_fopen(sysfs_path, "w");
+	if (!f) {
```

Comment:
Shouldn't we evaluate `errno` here to provide more targeted error messages?

Reply:
Yes, and we could merge it with the g_file_test above, which becomes redundant with this check.